Dockerfiles/mlpack-codespace/Dockerfile

```diff
@@ -0,0 +1,49 @@
+FROM mcr.microsoft.com/devcontainers/universal:focal
+
+LABEL maintainer="james.balamuta@gmail.com"
+
+## For apt to be noninteractive.
+ENV DEBCONF_NONINTERACTIVE_SEEN true
+
+## Install baseline dependencies not found in the devcontainer
+## For details on what's included, please see:
+## https://github.com/devcontainers/images/tree/main/src/universal
+RUN apt-get update -qq && apt-get install -y --no-install-recommends \
+    binutils-dev \ 
+    txt2man \
+    doxygen \
+    liblapack-dev \
+    libblas-dev \
+    libarpack2 \
+    libsuperlu-dev \
+    libstb-dev && \
+    apt-get clean && rm -rf /var/lib/apt/lists/* && \
+    pip3 --no-cache-dir install --upgrade --ignore-installed cython numpy \
+    pandas setuptools
+
+## Install armadillo.
+RUN curl -Lk https://files.mlpack.org/armadillo-11.4.1.tar.gz | tar -xvz && \
```

Comment:
Happy to put a newer version in place if you like.

Reply:
We'd want to bump the version associated with all images if that's the case. I think conrad just bumped the current default to: 10.8.2. So, we're potentially running one version higher than we should...

Reply:
That's only the minimum version, so I don't think it's a problem to have a newer one.  But if you want to keep it at the minimum, I put https://files.mlpack.org/armadillo-10.8.2.tar.gz in place.

Reply:
We probably should make sure everything is running on the minimums to avoid running into a hidden bug (even though the API is pretty stable).

Reply:
Switched it over to the minimum version.